### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by oehlschl** [jonbuda/miro#43]
It looks like there hasn't been a new official release of this gem since 2016; is the preferred method to reference github directly, or will a new release be cut soon? Came here in reference to the cocaine/terrapin deprecation.

Thanks in advance!

**Comment by jonbuda** [comment]
@oehlschl yes! I'm working on it. had to refactor a number of specs. But I'll get it out soon.

**Comment by idoo** [comment]
do you need any help?

**Comment by dcurtis** [comment]
@jonbuda Can you please update the rubygems.org release? Thanks!
